ddtrace/profiling/exporter/http.py

```diff
@@ -65,6 +68,19 @@ class PprofHTTPExporter(pprof.PprofExporter):
 
     endpoint_call_counter_span_processor = attr.ib(default=None, type=EndpointCallCounterProcessor)
 
+    def _update_git_metadata_tags(self, tags):
+        """
+        Update profiler tags with git metadata
+        """
+        # clean tags, because values will be combined and inserted back in the same way as for tracer
```

Comment:
I am not sure I follow this comment. What is wrong if `REPOSITORY_URL` is in `tags` (`DD_TAGS` I assume from the sounds of it?)